plans/features/steps.fmf

```diff
@@ -0,0 +1,33 @@
+summary: Verify functionality of individual steps
+description:
+    Test coverage focused on features related to execution of
+    individual steps.
+discover:
+    how: fmf
+    filter: "tier: 3"
+
+/discover:
+    discover+:
+        test: "^/tests/discover"
+
+/prepare:
+    discover+:
+        test: "^/tests/prepare"
+    prepare+:
+      - name: Refresh dnf cache to speed up execution
+        how: shell
+        script: |
+            podman run -itd --name fresh-fedora fedora
+            podman exec fresh-fedora dnf makecache
+            podman commit fresh-fedora fedora
+            podman run -itd --name fresh-centos centos:stream8
+            podman exec fresh-centos dnf makecache
+            podman commit fresh-centos centos:stream8
+
+/execute:
+    discover+:
+        test: "^/tests/execute"
+
+/the-rest:
+    discover+:
+        test: "^/tests/(provision|report|finish)"
```

Comment:
Out of curiosity, would a negative look ahead work here as well? `(?!discover|prepare|execute)`. It might be better, rather than listing steps to run, and possibly forgetting one, we would list steps we explicitly converted to standalone plans. The plan would then cover "the rest".

Reply:
Yes, works nicely: 16d30230

Reply:
Nice.